libs/ledgerjs/packages/cryptoassets/src/data/cardanoNative.json

```diff
@@ -37,7 +37,7 @@
     "SUNDAE",
     6,
     false,
-    false
+    true
```

Comment:
This sets `disableCountervalue` to `true`, is this expected? (same question for other tokens too)

Reply:
no it is not, I wasn't sure before so it was set to true for all. I'll update it